docs/sample.yml

```diff
@@ -491,6 +491,51 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  # Daiki
+  /courses/user:
+    post:
+      tags:
+        - courses
+      description: Add a user to a course
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                user_id:
+                  type: string
+                  description: |
+                    <span style="color: red; ">*required</span> A User's social account unique token id.
+                course_id:
+                  type: string
+                  description: |
+                    <span style="color: red; ">*required</span> A Unique course ID.
+                user_status:
+                  type: string
+                  description: |
+                    <span style="color: red; ">*required</span> A user's status for the course (course_id).<br>
+                    There are four status: <br>
+                    - wait list
+                      - A user who applied but not in the process because the course is not ready (the course is "inactive"). 
+                    - applied
+                      - A user who already applied but hasn't registered (got interview or paid tuition fee) yet.
+                    - registered
+                      - A user who already registered (got interview and paid tuition free) to the course. 
+                    - resign
+                      - A user who quit the class for some reason.
+              example:
+                  user_id: eyJz9sdfsdfsdfsd
+                  course_id: DPCS100#001
```

Comment:
course_id: DPCS100#001 -> course_id: DPCS100#1

Reply:
I changed the request parameters and it is fixed.